provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -3,6 +3,8 @@
 
 INTERNAL_IPS = ('127.0.0.1', '192.168.10.1',)
 
+ALLOWED_HOSTS = '*'
+
```

Comment:
Django docs state, "When DEBUG is True or when running tests, host validation is disabled; any host will be accepted. Thus it’s usually only necessary to set it in production."

Is this causing a problem somewhere?